data/Makefile.am

```diff
@@ -1,5 +1,11 @@
 SUBDIRS = desktop-files
 
+$(top_srcdir)/build/addin-xml-string-extractor.exe:
+	@pushd $(top_srcdir)/build;
+	make addin-xml-string-extractor.exe
+	popd;
```

Comment:
I'm not sure I understand why this is needed. If you run make with the existing Makefile.am stuff it compiles and creates addin-xml-string-extractor.exe already